src/arche/rules/result.py

```diff
@@ -120,3 +120,8 @@ def get_stats_count(self):
                 if message.stats is not None
             ]
         )
+
+    def show(self, short: bool = False, keys_limit: int = 10):
+        from arche.report import Report
```

Comment:
This thing concerns me, it is a circular dependency from `result.py` to `report.py`, which I temporaly circumvented by importing on a method level.

Reply:
What is the specific idea here?
Do you want to show only a specific result?

Reply:
Yep, the idea is to simplify using specific checks with chaining - `something_i_want_to_check(input).show()`